app/core/mypayment/endpoints_mypayment.py

```diff
@@ -97,6 +107,92 @@
 RETENTION_DURATION = 10 * 365  # 10 years in days
 
 
+@router.get(
+    "/mypayment/bank-account-information",
+    response_model=schemas_users.CoreUserSimple,
+    status_code=200,
+)
+async def get_bank_account_holder(
+    user: CoreUser = Depends(is_user()),
+    db: AsyncSession = Depends(get_db),
+):
+    """
+    Get the current bank account holder information.
+    """
+    bank_account_holder = await get_core_data(
+        MyPaymentBankAccountInformation,
+        db=db,
+    )
+    user_simple = schemas_users.CoreUserSimple(
+        id="",
+        name="",
+        firstname="",
+        account_type=AccountType.external,
+        school_id=uuid4(),
+    )
+    if bank_account_holder.holder_user_id != "":
+        user_db = await cruds_users.get_user_by_id(
+            user_id=bank_account_holder.holder_user_id,
+            db=db,
+        )
+        if user_db is None:
+            raise HTTPException(
+                status_code=404,
+                detail="Bank account holder user does not exist",
+            )
+        user_simple = schemas_users.CoreUserSimple(
+            id=user_db.id,
+            name=user_db.name,
+            firstname=user_db.firstname,
+            nickname=user_db.nickname,
+            account_type=user_db.account_type,
+            school_id=user_db.school_id,
+        )
+    return user_simple
+
+
+@router.post(
+    "/mypayment/bank-account-holder",
+    response_model=schemas_users.CoreUserSimple,
+    status_code=201,
+)
+async def set_bank_account_holder(
+    bank_account_info: MyPaymentBankAccountInformation,
+    db: AsyncSession = Depends(get_db),
+    user: CoreUser = Depends(is_user_in(GroupType.admin)),
+):
+    """Set the bank account holder information."""
+    if bank_account_info.holder_user_id == "":
+        raise HTTPException(
+            status_code=400,
+            detail="User ID cannot be empty",
+        )
```

Comment:
I don't this test is useful. If the id is empty, the following select will return None.